content/md/supplier-onboarder/dwl-product-import-tpl.md

```diff
@@ -17,3 +18,19 @@ Here is how to get them:
 1. Choose your preferred format: one file per family or one file with all families, in `CSV` or `XLSX`
 
 This template file can easily be imported in your Onboarder to add many products at once.
+
+## File format
+The template contains 3 lines in the header:
+1. attribute code, it allows the Onboarder to read the file when importing it
+1. attribute labels, it is the name of the attribute given by the retailer
```

Comment:
attribute labels: Human-readable designation of the attribute code above given by your retailer.